src/features/MovieList/MovieTile/styled.js

```diff
@@ -104,7 +110,7 @@ export const StyledStar = styled(Star)`
     }
 `;
 
-export const Rate = styled.p`
+export const Rate = styled.span`
     margin: 0px;
```

Comment:
Same here, units not needed, look through the file and check if it appears somewhere else.

Reply:
fixed